src/Cake.Common.Tests/Unit/Tools/OctopusDeploy/OctoPackTests.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Cake.Common.Tests.Fixtures.Tools;
+using Xunit;
+
+namespace Cake.Common.Tests.Unit.Tools.OctopusDeploy
+{
+    public sealed class OctoPackTests
+    {
+        public sealed class ThePackMethod
+        {
+            [Fact]
+            public void Should_Throw_If_Id_Is_Null()
+            {
+                var fixture = new OctopusDeployPackerFixture();

```

Comment:
We normally divides the Arrange,Act,Assert in `Given`/`When`/`Then` comments.
Would this be possible to change?

```csharp
public void Test()
{
  // Given
  var fixture = new MagicFixture();

  // When
  var result = fixture.DoMagic();

  // Then
  Assert.Equal(42, result);
}